website/pages/explore/dir-content/relational-database.asciidoc

```diff
@@ -0,0 +1,32 @@
+= RELATIONAL DATABASE
+
+[.directory]
+== Relational Database
+
+[.links-to-files]
+== Related
+
+<<oracle.html#, Oracle>>
+
+[.common-links]
+=== Links
+
+[.devon4j-links]
+=== devon4j Links
+
+<</website/pages/docs/guide-database.asciidoc.html#guide-database.asciidoc_rdbms, Database Guide>>
+
+[.devon4net-links]
+=== devon4net Links
+
+https://docs.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli
```

Comment:
this isn't really a devon4net link, more an external one.
Let's remove that as such links should better incorporated into the docs section if they are needed.